packages/trpc/server/routers/viewer/eventTypes.ts

```diff
@@ -647,27 +646,6 @@ export const eventTypesRouter = router({
       };
     }
 
-    if (input?.price || input.metadata?.apps?.stripe?.price) {
-      data.price = input.price || input.metadata?.apps?.stripe?.price;
-      const paymentCredential = await ctx.prisma.credential.findFirst({
-        where: {
-          userId: ctx.user.id,
-          type: {
-            contains: "_payment",
-          },
-        },
-        select: {
-          type: true,
-          key: true,
-        },
-      });
-
-      if (paymentCredential?.type === "stripe_payment") {
-        const { default_currency } = stripeDataSchema.parse(paymentCredential.key);
-        data.currency = default_currency;
-      }
-    }
-
```

Comment:
Since we rely on payment app data, I don't think we need this as a part of the event type update function. 

Payment apps aren't working on team event types (yet) and the only place we use `eventType.price` is when converting legacy event types.